14102025-esrin-rome-italy/prep-work/readme.md

```diff
@@ -0,0 +1,60 @@
+# About
+
+This directory is a workspace to collaborate and prepare for the STAC sprint. Everyone is welcome
+to edit files and make new workspaces - just make a PR and we will get your information into main. Work will also take place
+in other repositories that are specific to the STAC specifications or ecosystem; this space just serves as a collaboration space
+with a low barrier to entry.
+
+# Meetings to Attend
+
+There are a handful of STAC meetings occurring in the time leading up to the STAC. We encourage STAC sprint attendees to make as many of the upcoming meetings as possible to get everyone as up-to-speed as possible before the sprint.
+
+#### STAC Community Meeting
+
+General meeting about all things stac-spec and stac-api-spec. The meeting consists of (1) intros and updates from each meeting participant and (2) agenda items (created by anyone that attends the meeting).
+
+**Meeting time:** Every other Monday from 11 am - 12 pm EST (1-hour long meeting)
+
+- Monday, September 8th @ 11 am EST
+
+- Monday, September 22 @ 11 am EST
+
+- Monday, October 6th @ 11 am EST
+
+#### STAC Working Session
+
+A two-hour meeting that allows us to dive into a specific task and get some work done while all online. The agenda is preset by the STAC PSC and there are no intros/updates/check-ins.
+
+**Meeting time:** 1st Tuesday of the month from 11 am - 1 pm EST (2-hour long meeting)
+
+- Tuesday, September 2nd @ 11 am EST
+
+#### STAC Ecosystem (stac-utils) Meeting
+
+These meetings are relatively new but have proven helpful to the community. The format of this meeting is similar to that of the STAC Community meeting, but instead of discussing the specifications themselves (stac-spec and stac-api-spec), this meeting discusses work being done/needed for the tools built to help interact with STAC.
+
+**Meeting time:** occasional Mondays from 11 am - 12 pm EST (1-hour long meeting)
+
+- ?
```

Comment:
I think these are not happening anymore?!

Reply:
You are correct @m-mohr !